app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,15 +1,19 @@
 module Api
   module V1
     class RentsController < ApiController
-      include Wor::Paginate
+      rescue_from Pundit::NotAuthorizedError, with: :not_authorized
```

Comment:
Try doing this rescue from the api controller, that way every controller will have this rescue_from